mmdeploy/codebase/mmdet/models/dense_heads/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .reppoints_head import reppoints_head__predict_by_feat
 from .rpn_head import rpn_head__get_bboxes__ncnn, rpn_head__predict_by_feat
 from .rtmdet_head import rtmdet_head__predict_by_feat
+from .solo_head import solohead__predict_by_feat__default
```

Comment:
fix conflicts with latest dev-1.x and change the style of importing rewriting functions.